doc/debian/simbody.install

```diff
@@ -0,0 +1 @@
+usr/libexec/*
```

Comment:
@j-rivero I think the `simbody-visualizer` executable should be included in the `libsimbody3.6` package.

Reply:
umm that is an interesting point. According to the [FHS libexec](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s07.html) is used by binaries designed not to call directly by the user. Is this case? If it is, then I think that you are right. If we want to invoke the binaries directly, we probably need to move them to bin/ and use the simbody package.